### PR TITLE
Adding the flake8 comment to the PR template

### PR DESCRIPTION
**Pull request by bsipocz**
cc @tomdonaldson 

Would this be useful? The difficult task will be to keep the command up-to-date, but I hope we can do that.

### CODE REVIEW

**Comment by astrofrog** [comment]
> The difficult task will be to keep the command up-to-date

Not if we just fix all the PEP8 issues 😉

**Comment by bsipocz** [comment]
> Not if we just fix all the PEP8 issues 

I wouldn't be against that!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ http://docs.astropy.org/en/latest/development/workflow/development_workflow.html
 There is even a practical example at
 https://docs.astropy.org/en/latest/development/workflow/git_edit_workflow_examples.html#astropy-fix-example . -->
 
+<!-- Astropy coding style guidelines can be found here:
+https://docs.astropy.org/en/latest/development/codeguide.html#coding-style-conventions
+Our testing infrastructure enforces to follow a subset of the PEP8 to be
+followed. You can check locally whether your changes have followed these by
+running the following command:
+
+flake8 astropy --count -select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823
```
Comment (tomdonaldson) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
flake8 astropy --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823
```

**Comment by Cadair** [comment]
Can't we have this command pick up the configs in `setup.cfg` rather than having to specify it all on the CLI?

**Comment by pllim** [comment]
@Cadair , I *think* the idea is that people can use the `--select` to see the warnings but we are not enforcing it via `setup.cfg`. Maybe @bsipocz can clarify if I misunderstood.

**Comment by saimn** [comment]
Yes, if this was in `setup.cfg` then you would miss other useful warnings in your editor, because this list of warnings is only a subset for the CI build.

**Comment by bsipocz** [comment]
@Cadair - the issue with adding this to setup.cfg is that all the rest of the pep8 was ignored in our editors. Maybe we can do the setup.cfg way once we switch to ignore a small number of warnings rather than select a few.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (bsipocz) [reply .github/PULL_REQUEST_TEMPLATE.md]: indeed, messed up the  copy paste...

**Comment by bsipocz** [comment]
Going  ahead and merge  this,  CI  doesn't need to run.

**Pull request merged**